layers/meta-balena-up-board/recipes-kernel/linux/linux-yocto_%.bbappend

```diff
@@ -1,5 +1,20 @@
 inherit kernel-resin
 
+# HDMI audio support requested by customer
+RESIN_CONFIGS_append = " hdmi_sound"
+RESIN_CONFIGS[hdmi_sound] = " \
+    CONFIG_SND_HDA_INTEL=m \
+"
+
+# ATH10K support requested by customer on Up Squared
+RESIN_CONFIGS_append = " ath10k_pci"
+RESIN_CONFIGS[ath10k_pci] = " \
+    CONFIG_ATH10K_PCI=m \
+"
+RESIN_CONFIGS_DEPS[ath10k_pci] = " \
+    CONFIG_ATH10K=m \
+"
+
 # requested by customer
 RESIN_CONFIGS_append = " ch341"
 RESIN_CONFIGS[ch341] = " \
```

Comment:
Maybe ch341 could be dropped from there now too? https://github.com/balena-os/meta-balena/pull/1573/commits/416200a2e49a146d8de20791a782e91b94aa49cd

Reply:
We'll probably do a release for the customer before meta-balena gets released